### PR TITLE
Add PTS performance builder

### PR DESCRIPTION
**Pull request by behlendorf**
* Add a bb-test-pts which runs several of the PTS "Disk" tests
  for four different zpool configurations on a dedicated
  performance builder.

* Initial version of test profiles to run for ZFS described by
  the pts-test-profiles.tar.gz tarball.  The exact performance
  values here aren't as important as our relative delta over time.

### CODE REVIEW

File: scripts/bb-test-cleanup.sh [hunk 1]
```diff
@@ -177,6 +177,9 @@ export CODE_COVERAGE_BRANCH_COVERAGE=${TEST_CODE_COVERAGE_BRANCH_COVERAGE:-0}
 set -x
 cd "${ZFS_BUILD}"
 
+# Unload modules
+sudo -E $ZFS_SH -vu
```
Comment (dinatale2) [inline scripts/bb-test-cleanup.sh]: Is this necessary anymore? The PERF builder doesn't run cleanup anymore.

File: master/master.cfg [hunk 2]
```diff
@@ -1530,6 +1540,15 @@ c['schedulers'].append(Nightly(
     onlyIfChanged=False,
     dayOfWeek=6, hour=19, minute=0))
 
+# This scheduler is to trigger nightly performance tests.
+#c['schedulers'].append(Nightly(
+#    name="nightly-pts-scheduler",
```
Comment (dinatale2) [inline master/master.cfg]: Was this intentionally commented out?

Comment (behlendorf) [reply master/master.cfg]: Nope, fill fix.

File: scripts/bb-test-cleanup.sh (hunk 1 above)
Comment (behlendorf) [reply scripts/bb-test-cleanup.sh]: Right, I'll drop it.

**Comment by behlendorf** [comment]
@dinatale2 I'm not sure.  That's something we'll want to investigate and may need a follow up PR to address.

**Comment by dinatale2** [comment]
@behlendorf Sounds good.

**Pull request merged**